tests/Core/Log/Context/CancellationRequestIdContextTest.php

```diff
@@ -0,0 +1,37 @@
+<?php declare(strict_types=1);
+/**
+ * This File is part of JTL-Software
+ *
+ * User: avermeulen
+ * Date: 2021-01-22
+ */
+
+namespace Core\Log\Context;
+
+use JTL\SCX\Lib\Channel\Core\Log\Context\CancellationRequestIdContext;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers \JTL\SCX\Lib\Channel\Core\Log\Context\CancellationRequestIdContext
+ */
+class CancellationRequestIdContextTest extends TestCase
+{
+    public function testCanBeUsed(): void
```

Comment:
Bitte gib den Test einen ordentlichen namen - sag was du testest.

`it will add orderCancellationRequestId to log record array`